### PR TITLE
carousal added

### PR DESCRIPTION
**Pull request by Chetan8299**
added carousal for fertilizers, agriculture machines and seeds

### CODE REVIEW

**Comment by desujoy** [comment]
@Chetan8299 check build logs and fix it

**Comment by shreyvarshney1** [comment]
This has failed checks. Check deployment otherwise it will be closed.

File: app/page.tsx
Comment (desujoy) [inline app/page.tsx:1]: move constants outside page

File: components/carousal/slidersample.css
Comment (desujoy) [inline components/carousal/slidersample.css:1]: try to use css variables from `global.css`

**Review by desujoy — CHANGES_REQUESTED** [review]
do the changes requested

File: app/page.tsx
Comment (desujoy) [inline app/page.tsx:1]: put carousal before newsletter `SubscribeEmailInput`

**Review by desujoy — CHANGES_REQUESTED** [review]
`SwiperComponent` is modified by css files try to use .module.css or tailwind

**Comment by shreyvarshney1** [comment]
@Chetan8299 please complete these changes so that we can merge this pr

**Review by shreyvarshney1 — CHANGES_REQUESTED** [review]
fix css

**Comment by desujoy** [comment]
@Chetan8299  main branch updated run `git merge main` before your next commit

**Comment by shreyvarshney1** [comment]
open a new pr @Chetan8299

**Pull request closed without merging**